db/migrate/20150628060637_sorcery_external.rb

```diff
@@ -0,0 +1,12 @@
+class SorceryExternal < ActiveRecord::Migration
+  def change
+    create_table :authentications do |t|
+      t.integer :user_id, :null => false
+      t.string  :provider, :uid, :null => false
```

Comment:
Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Put one space between the method name and the first argument.